source/common/router/config_impl.h

```diff
@@ -387,6 +387,7 @@ class RetryPolicyImpl : public RetryPolicy {
   ProtobufMessage::ValidationVisitor* validation_visitor_{};
   std::vector<Upstream::RetryOptionsPredicateConstSharedPtr> retry_options_predicates_;
 };
+using DefaultRetryPolicy = ConstSingleton<RetryPolicyImpl>;
```

Comment:
nit: I think this and the other aliases can be moved into the `RouteEntryImplBase` class, to decrease namespace pollution.